src/components/student/Transcript.js

```diff
@@ -1,18 +1,102 @@
-import React, {useState, useEffect} from 'react';
+import React, { useState, useEffect } from 'react';
+import { SERVER_URL } from '../../Constants';
 
-// students gets a list of all courses taken and grades
-// use the URL /transcript?studentId=
-// the REST api returns a list of EnrollmentDTO objects 
-// the table should have columns for 
-//  Year, Semester, CourseId, SectionId, Title, Credits, Grade
+const Transcript = () => {
+    const [enrollments, setEnrollments] = useState([]);
+    const [message, setMessage] = useState('');
+    const studentId = 3;
 
-const Transcript = (props) => {
-     
-    return(
-        <> 
-            <h3>Not implemented</h3>
-        </>
+    useEffect(() => {
+        fetchTranscript();
+    }, []);
+
+    const fetchTranscript = async () => {
+        try {
+            const response = await fetch(`${SERVER_URL}/transcript?studentId=${studentId}`);
+            if (response.ok) {
+                const data = await response.json();
+                setEnrollments(data);
+            } else {
+                const errorData = await response.json();
+                setMessage(`Error: ${errorData.message}`);
+            }
+        } catch (error) {
+            setMessage(`Network error: ${error.message}`);
+        }
+    };
```

Comment:
This was pretty much the same implementation as the other branch in the end, although I did like the formatting a little more. We will just stay with v1.0 branch.

Reply:
sounds good we can stick with the v1.0 branch.